### PR TITLE
Add Navigation Bar Logo

### LINKED ISSUE #98: Nav Bar UI - Logo
NOTE: This story does NOT cover the Filter button

* Logo with circular icon + "FightPandemics" title

Screen Shot 2020-05-17 at 4 24 35 PM

### PR DESCRIPTION
**Pull request by OnyekachiSamuel**
**What this PR does**:
This PR adds a navigation bar to the navbar. It also positions the filter button rightly the way we have it in the design.
### Checklist

- [✅] Compiler warnings resolved
- [✅] Linter warnings resolved
- [✅] User-facing strings in Localizable.strings file
- [✅] Unused code, print statements, and comments removed

**Which issue(s) this PR fixes**:
#98 

**Screenshot**
Simulator Screen Shot - iPhone 11 - 2020-05-20 at 13 53 52

**Special notes for reviewers**:
The profile screen also has the navbar logo, when the design for the profile screen is done, we can then make the necessary adjustment if there is need for it.

### CODE REVIEW

**Review by ethanswift — APPROVED** [review]

**Review by stavares843 — COMMENTED** [review]
develop - https://jmp.sh/4Qhu57h
this branch - https://jmp.sh/1yJxuNJ

the nav bar should be a bit higher

**Review by stavares843 — COMMENTED** [review]
also on figma doesnt have the separator line below the nav bar

https://jmp.sh/BhVz2Cf

File: FightPandemics/Utilities/Extensions/UINavigationBar+FightPandemics.swift [hunk 1]
```diff
@@ -27,6 +27,24 @@
 import UIKit
 
 extension UINavigationBar {
+    /// Configures the navigation bar appearance
+    func configure() {
+        if #available(iOS 13.0, *) {
```
Comment (haidalinda) [inline FightPandemics/Utilities/Extensions/UINavigationBar+FightPandemics.swift]: we're targeting `13.0+` - no need for the conditional

**Comment by haidalinda** [comment]
Screen Shot 2020-05-20 at 9 06 53 PM

pushing on top of the navigation stack looks odd 😖 (example: Profile > OSS)

**Review by haidalinda — CHANGES_REQUESTED** [review]
see commends

File: FightPandemics/Utilities/Extensions/UINavigationBar+FightPandemics.swift (hunk 1 above)
Comment (OnyekachiSamuel) [reply FightPandemics/Utilities/Extensions/UINavigationBar+FightPandemics.swift]: Okay, cool, I will change that

**Comment by haidalinda** [comment]
feel free to move the `Log Out` button on `Profile` into the body - its messing with your look

**Review by haidalinda — APPROVED** [review]

**Pull request merged**